src/snowflake/cli/plugins/spcs/image_repository/manager.py

```diff
@@ -60,3 +70,12 @@ def get_repository_api_url(self, repo_url):
         path = parsed_url.path
 
         return f"{scheme}://{host}/v2{path}"
+
+    def _remove_scheme(self, url: str) -> str:
+        if not urlparse(url).scheme and not url.startswith("//"):
+            url = f"//{url}"
+        parsed_url = urlparse(url)
+        return f"{parsed_url.netloc}{parsed_url.path}"
+
+    def get_repository_url_strip_scheme(self, repo_name):
```

Comment:
I am not sure what does strip_schema mean. Would you mind adding some comments of reasons and input url, output url examples? Why don't we use the `get_repository_row` result directly?